src/actions/agreement/bill.js

```diff
@@ -62,27 +64,27 @@ const freeAgreementForUser = (username) => ({
   },
 });
 
-const kBannedStates = ['cancelled', 'suspended'];
+const bannedStates = ['cancelled', 'suspended'];
```

Comment:
```suggestion
const kBannedStates = ['cancelled', 'suspended'];
```
тут корректнее k (это как в C конвенциях константы) или выкинуть так же в constants

Reply:
пофиксил